### PR TITLE
[fr] FR translations

### PR DESCRIPTION
**Pull request by HavokInspiration**
Follows #4994, #4995, #5002, #5004, #5018, #5020, #5022, #5026, #5030, #5039, #5042, #5047, #5055.

As usual, I may have done some small modifications to the original version.

### CODE REVIEW

File: fr/controllers/components/security.rst [hunk 1]
```diff
@@ -148,7 +151,8 @@ validatePost
     essentiellement éteindre la validation de formulaire.
 
 Les options de configuration ci-dessus peuvent être _set_ via la méthode
-``config()``
+``setConfig()`` ou ``config()`` si vous utilisez une version de CakePHP avant
```
Comment (antograssiot) [inline fr/controllers/components/security.rst]: `inférieure à` au lieu de `avant` ? (Ici et en dessous)

File: fr/orm/validation.rst [hunk 2]
```diff
@@ -29,6 +29,9 @@ retournée::
         // validation de l'entity a échouée.
     }
 
+.. versionadded:: 3.4.0
+    LA méthode ``getErrors()`` a été ajoutée.
```
Comment (antograssiot) [inline fr/orm/validation.rst]: La

**Comment by HavokInspiration** [comment]
@antograssiot Fixed pushed for both comments.

**Comment by antograssiot** [comment]
merci Yves 👍

**Pull request merged**